File: Backend/Nodejs/src/models/database/connection.ts
import { createConnection } from "mysql";

export const pool = createConnection({
  host: process.env.DB_HOST,
  user: process.env.DB_USER,
  password: process.env.DB_PASSWORD,
  database: process.env.DB_DB,
});
pool.connect(function (err) {
  if (err) {
    console.error("error connecting: " + err.stack);
    return;
  }
  console.log("connected as id " + pool.threadId);
});

File: Backend/Python/src/models/database/connection.py
import mysql.connector
from mysql.connector import errorcode
from decouple import config

cnx = None


def connect():
    try:
        global cnx
        cnx = mysql.connector.connect(
            user=config("DB_USER"),
            password=config("DB_PASSWORD"),
            host=config("DB_HOST"),
            database=config("DB_DB"),
        )
        print(cnx)
    except mysql.connector.Error as err:
        if err.errno == errorcode.ER_ACCESS_DENIED_ERROR:
            print("Something is wrong with your user name or password")
        elif err.errno == errorcode.ER_BAD_DB_ERROR:
            print("Database does not exist")
        else:
            print(err)


def getCnx():
    return cnx
